### DISCUSSION TITLE
How to get the Heart On Your Sleeve Badge

### DISCUSSION THREAD

**Discussion by dev-aniketj** [community/community#44971] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

How should I get the Heart On Your Sleeve Badge
![image](https://user-images.githubusercontent.com/81229551/213864014-d782eb92-d1ef-4eca-bcf9-9367850eb15e.png)

**Comment by canaleal** [comment, marked as answer]
Hey @dev-aniketj, unfortunately it is currently unknown how to get the Heart On Your Sleeve Badge.

As mentioned by the [Github introducing the achievement badges](https://github.blog/2022-06-09-introducing-achievements-recognizing-the-many-stages-of-a-developers-coding-journey/), they said that they will never tell how many badges there are and how to unlock them.

The best way to find out how to get the badge is to wait for someone else to get it and then announce how they got it. 
You could use this [unofficial repository](https://github.com/drknzz/GitHub-Achievements) that lists all the achievements to see how to get the badge.


Hope this clarifies & answers your question

**Reply by NadavIs56** [reply]
❤️

**Reply by DeveloperrEva** [reply]
❤️

**Reply by Ismailvk** [reply]
❤️

**Reply by Flumsykun** [reply]
❤️

**Reply by mdjeghbala** [reply]
❤️

**Reply by brian09088** [reply]
❤️

**Comment by jge162** [comment]
I don’t think it’s enabled atm. Haven’t seen it online anywhere.

**Reply by dev-aniketj** [reply]
Nope

**Comment by Tynab** [comment]
To earn the Heart on Your Sleeve achievement on GitHub, you need to publicly share a project on GitHub that contains a valid license and a README file. Here are the steps you can follow:
1. Sign up for a GitHub account if you haven't already.
2. Create a new project or select an existing project that you want to share on GitHub. The project should include a valid open source license and a README file that provides information about the project.
3. Create a new repository on GitHub and upload your project files.
4. Add a description and any other relevant information about your project, such as installation instructions or usage examples, to the README file.
5. Choose an appropriate open source license for your project and include a LICENSE file in the repository. GitHub provides several options for open source licenses that you can choose from.
6. Make sure that the repository is set to "public" so that others can view and access your project.
7. After you have completed these steps, you should receive the Heart on Your Sleeve badge on your GitHub profile.
Note that GitHub has specific requirements for what constitutes a valid open source license, so be sure to review the license options and choose one that fits the needs of your project. Additionally, it's important to provide clear and comprehensive information about your project in the README file to help others understand how to use and contribute to your project. Good luck!

**Reply by consciousness425** [reply]
@Tynab 
If you are right, why did not you get yet?

**Reply by moritztim** [reply]
I hate that people are just automating github discussions with AI. And it's not even working

**Comment by github-staff** [comment]
Thanks so much for posting in our Community @dev-aniketj. Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀
